CONTRIBUTING.md

```diff
@@ -278,7 +286,7 @@ An entry in `bin` needs to be in `PATH` identify with `which external_dependency
 An entry in `python` needs to be in `PYTHONPATH` identify with `python -c "import external_dependency_python_N"` command.
 
 #### ImportError
-In python files where you use a `import external_dependency_python_N` you will need to add a `try-except` with a info log.
+In python files where you use a `import external_dependency_python_N` you will need to add a `try-except` with a debug log.
```

Comment:
@yvaucher 
You are right.

First reason here:
![local](https://cloud.githubusercontent.com/assets/6644187/9636503/58b8ab4a-5161-11e5-9cd4-cc27566b5da0.png)